### PR TITLE
dsw-1281: add icons test pages to vanilla and nextjs apps

### PR DESCRIPTION
**Pull request by jamieomaguire**
Adds an icons page to the aperture nextjs and vanilla apps. We need to wait for the react wrapper PIE to be merged into PIE first so we can update to the newer icons package version before this is merged.

## Next JS app
Screenshot 2023-11-24 at 18 09 31

## Vanilla App
Screenshot 2023-11-24 at 18 10 09

### CODE REVIEW

**Review by ashleynolan — APPROVED** [review]

File: nextjs-app/package.json [hunk 1]
```diff
@@ -16,18 +16,20 @@
...
     "@lit-labs/nextjs": "0.1.4",
     "@lit/react": "1.0.2",
+    "add": "^2.0.6",
     "lit": "3.1.0",
     "next": "13.5.6",
     "react": "18.2.0",
     "react-dom": "18.2.0",
-    "sass": "1.69.5"
+    "sass": "1.69.5",
+    "yarn": "^1.22.21"
```
Comment (siggerzz) [inline nextjs-app/package.json:32]: yarn probably shouldn't be added as dev dep. Was this done for any reason in particular? I notice we're using Yarn 3 at the root. - https://github.com/justeattakeaway/pie-aperture/tree/main/.yarn/releases

Comment (jamieomaguire) [reply nextjs-app/package.json:32]: I have absolutely no idea how that got there - will remove now! Nice spot

**Review by siggerzz — APPROVED** [review]

File: nextjs-app/src/pages/index.tsx [hunk 2]
```diff
@@ -6,9 +6,10 @@ import { PieSpinner } from "@justeattakeaway/pie-spinner/dist/react"
 import { PieModal } from "@justeattakeaway/pie-modal/dist/react"
 import { PieCookieBanner } from "@justeattakeaway/pie-cookie-banner/dist/react"
 import { PieIconButton } from "@justeattakeaway/pie-icon-button/dist/react"
-import '@justeattakeaway/pie-icons-webc/IconClose';
+import { IconClose } from "@justeattakeaway/pie-icons-webc/dist/react/IconClose";
 import { PieSwitch } from "@justeattakeaway/pie-switch/dist/react"
 import { PieCard } from "@justeattakeaway/pie-card/dist/react"
+import Link from 'next/link'
```
Comment (xander-marjoram) [inline nextjs-app/src/pages/index.tsx:12]: Semicolon police 🚓

**Review by xander-marjoram — APPROVED** [review]

File: nextjs-app/src/pages/index.tsx (hunk 2 above)
Comment (jamieomaguire) [reply nextjs-app/src/pages/index.tsx:12]: Let's add a ticket to set up linting in general!

**Pull request merged**